doc/impl.md

```diff
@@ -1,7 +1,7 @@
 ## Files
 
 The implementation of leveldb is similar in spirit to the representation of a
-single [Bigtable tablet (section 5.3)](http://research.google.com/archive/bigtable.html).
+single [Bigtable tablet (section 5.3)](https://research.google/pubs/pub27898/).
```

Comment:
Verified that `http://research.google.com/archive/bigtable.html` redirects to `https://research.google/pubs/pub27898/`.